rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/rename/Common.rsc

```diff
@@ -53,22 +54,41 @@ data RenameConfig(
   , PathConfig(loc) getPathConfig = PathConfig(loc l) { throw "No path config for <l>"; }
 );
 
-bool(loc) containsFilter(type[&T <: Tree] t, str name, str(str) escape, Tree(loc) getTree) {
-    Tree n = parse(t, name);
-    Tree en = parse(t, escape(name));
-    return bool(loc l) {
-        bottom-up-break visit (getTree(l)) {
-            case n: return true;
-            case en: return true;
-        }
-        return false;
-    };
+bool(loc) containsFilter(type[&T <: Tree] T, str name, Tree(loc) getTree) {
+    if (just(t) := tryParseAs(T, name)) {
+        return bool(loc l) {
+            bottom-up-break visit (getTree(l)) {
```

Comment:
nit: since we return on the first match, `bottom-up` is enough, and thats already the standard.